src/librustc_parse/parser/mod.rs

```diff
@@ -671,12 +671,12 @@ impl<'a> Parser<'a> {
                 true
             }
             token::BinOp(token::Shl) => {
-                let span = self.token.span.with_lo(self.token.span.lo() + BytePos(1));
```

Comment:
I see a bunch of other instances of `self.token.span.lo() + BytePos(1)` in this file. Should they be updated too?

Reply:
I'm fine to do so if @estebank also agrees with that.

Reply:
Uhm, I found out it shouldn't, it'll break many spans.

Reply:
What kind of breakage occurs?

Reply:
Indeed we can use `next_ponit` for `eat_plus` but it seems to cause breakage in other places (`expect_and`, `expect_or`, and `expect_gt`). It will expand or reduce current span size.

Reply:
I guess we should also tweak later `bump_with`s?

Reply:
Can you file a ticket to follow up on this?

Reply:
Filed as #68783.